docs/pages/reference/react/types.mdx

```diff
@@ -112,10 +113,17 @@ export type FrameButtonPostRedirectProvidedProps = {
   children: string | number;
 };
 
+export type FrameButtonMintProvidedProps = {
+  /** a label to display on the button */
+  children: string | number;
+  action?: "mint";
+  /** a [CAIP-10](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-10.md) url to an NFT smart contract or token */
+  mint: string;
```

Comment:
how do you feel about a stricter type here - could annoy devs but our helpers could return it as well

Reply:
i had a stricter type initially but it was really annoying. i think adding validation to the debugger would be more helpful